packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -195,7 +195,13 @@ class DatePickerInput extends React.Component {
 
     const label = this.props.ariaLabel ? this.props.ariaLabel : intl.formatMessage({ id: 'Terra.datePicker.date' });
 
-    this.formatDescriptionId = (useExternalFormatMask === false) ? `terra-date-picker-description-format-${this.uuid}` : '';
+    if (useExternalFormatMask === false) {
+      this.formatDescriptionId = `terra-date-picker-description-format-${this.uuid}`;
+    } else if (inputAttributes['aria-describedby']) {
```

Comment:
why we need `inputAttributes['aria-describedby']` as an Id here...??? since [format-div ](https://github.com/cerner/terra-framework/blob/ee1e2890c5b5f39b5966708110c07e68a27b05b7/packages/terra-date-picker/src/DateInput.jsx#L246) will only be rendered when  `useExternalFormatMask` is false.

Reply:
It is there for [aria-describedby](https://github.com/cerner/terra-framework/blob/ee1e2890c5b5f39b5966708110c07e68a27b05b7/packages/terra-date-picker/src/DateInput.jsx#L229) in Input to read the help and error message coming from datepickerfield.

If you see the latest version, the voiceover doesn't read the help and error message in datepickerfield. That is due to typo [here](https://github.com/cerner/terra-framework/blob/main/packages/terra-date-picker/src/DatePickerField.jsx#L270) in

Reply:
Do you have a small video clip of the current vs fixed behavior that demonstrates this?

Reply:
Should `useExternalFormatMask` have higher precedence over `aria-describedby` or the other way around? `useExternalFormatMask` will always be false when not using DatePickerField and therefore, it will always use the uuid and ignores `aria-describedby` even if it is provided.

Reply:
Added a check for when both are true here https://github.com/cerner/terra-framework/pull/1252/commits/ef5efe7b86697d7565cee87caa635627f42c26b1.